### PR TITLE
aosp.rst: update doc to reflect current dev env

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: building/prerequisites.rst [hunk 1]
```diff
@@ -16,5 +16,5 @@ target you will use in the end.
             expect flex ftp-upload gdisk iasl libattr1-dev libc6:i386 libcap-dev \
             libfdt-dev libftdi-dev libglib2.0-dev libhidapi-dev libncurses5-dev \
             libpixman-1-dev libssl-dev libstdc++6:i386 libtool libz1:i386 make \
-            mtools netcat python-crypto python3-pyelftools python-serial \
+            mtools netcat python-crypto python3-pyelftools python-serial python-wand \
```
Comment (jbech-linaro) [inline building/prerequisites.rst]: python-wand is not needed I guess? I think we used that when we had the TUI prototype in OP-TEE?

Comment (vchong) [reply building/prerequisites.rst]: Ok. It was in `aosp.rst` so included it here. Will remove it.

**Comment by vchong** [comment]
Updated

**Comment by vchong** [comment]
Tag applied. Ok for merge. Thanks!

**Pull request merged**